source/pages/Home/Panel/components/AssetDetails.tsx

```diff
@@ -80,7 +75,12 @@ export const AssetDetails = ({ assetType, assetData }) => {
 
   return (
     <>
-      {imageLink && !loadingImage ? (
+      {hasImage && !imageLink && (
```

Comment:
we need to show an image in any case because this is a new feature in pali and the users would know, even if there is no image, that pali can show the image token, so it is required to show a placeholder if there is no image